Problems building samples on OS X

I had some problems getting the following samples to compile on OS X 10.11.1 with Xcode 7.1.1:
* ocvColorQuantize
* ocvFaceDetect 
* ocvOpticalFlow
* ocvPerspective
* ocvWarp

They all had problems finding the boost libs:
```
../../../../../include/cinder/Filesystem.h:35:11: error: 'boost/filesystem.hpp' file not found with <angled> include; use "quotes" instead
        #include <boost/filesystem.hpp>
                 ^~~~~~~~~~~~~~~~~~~~~~
                 "boost/filesystem.hpp"
In file included from /Users/amorton/projects/Cinder/blocks
```

It seems like the fix for that is to change the "Header Search Path" to `"$(CINDER_PATH)/include"`. 

~~Changing that fixed all but the ocvFaceDetect sample which still had another error:~~
```
Undefined symbols for architecture x86_64:
  "cinder::app::Platform::getAssetPath(boost::filesystem::path const&) const", referenced from:
      cinder::app::AppBase::getAssetPath(boost::filesystem::path const&) const in ocvFaceDetectApp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
~~I'm not quite sure what to do about that.~~

--- a/samples/ocvBasic/xcode_ios/ocvBasic.xcodeproj/project.pbxproj
+++ b/samples/ocvBasic/xcode_ios/ocvBasic.xcodeproj/project.pbxproj
@@ -361,7 +361,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvBasic_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";
@@ -375,7 +375,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvBasic_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvBasic/xcode_ios/ocvBasic_Prefix.pch
+++ b/samples/ocvBasic/xcode_ios/ocvBasic_Prefix.pch
@@ -14,8 +14,6 @@
 
 #include "cinder/Cinder.h"
 
-#include "cinder/app/AppCocoaTouch.h"
-
 #include "cinder/gl/gl.h"
 
 #include "cinder/CinderMath.h"

--- a/samples/ocvCapture/xcode_ios/ocvCapture.xcodeproj/project.pbxproj
+++ b/samples/ocvCapture/xcode_ios/ocvCapture.xcodeproj/project.pbxproj
@@ -333,7 +333,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvCapture_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";
@@ -347,7 +347,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvCapture_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvColorQuantize/xcode/ocvColorQuantize.xcodeproj/project.pbxproj
+++ b/samples/ocvColorQuantize/xcode/ocvColorQuantize.xcodeproj/project.pbxproj
@@ -312,7 +312,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
@@ -329,7 +329,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				SDKROOT = macosx;
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvColorQuantize/xcode_ios/ocvColorQuantize.xcodeproj/project.pbxproj
+++ b/samples/ocvColorQuantize/xcode_ios/ocvColorQuantize.xcodeproj/project.pbxproj
@@ -309,7 +309,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvColorQuantize_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";
@@ -323,7 +323,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvColorQuantize_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvFaceDetect/xcode/ocvFaceDetect.xcodeproj/project.pbxproj
+++ b/samples/ocvFaceDetect/xcode/ocvFaceDetect.xcodeproj/project.pbxproj
@@ -316,7 +316,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
@@ -333,7 +333,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				SDKROOT = macosx;
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvOpticalFlow/xcode/ocvOpticalFlow.xcodeproj/project.pbxproj
+++ b/samples/ocvOpticalFlow/xcode/ocvOpticalFlow.xcodeproj/project.pbxproj
@@ -316,7 +316,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
@@ -333,7 +333,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				SDKROOT = macosx;
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvOpticalFlow/xcode_ios/ocvOpticalFlow.xcodeproj/project.pbxproj
+++ b/samples/ocvOpticalFlow/xcode_ios/ocvOpticalFlow.xcodeproj/project.pbxproj
@@ -307,7 +307,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvOpticalFlow_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";
@@ -321,7 +321,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvOpticalFlow_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvPerspective/xcode/ocvPerspective.xcodeproj/project.pbxproj
+++ b/samples/ocvPerspective/xcode/ocvPerspective.xcodeproj/project.pbxproj
@@ -316,7 +316,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
@@ -333,7 +333,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				SDKROOT = macosx;
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvPerspective/xcode_ios/ocvPerspective.xcodeproj/project.pbxproj
+++ b/samples/ocvPerspective/xcode_ios/ocvPerspective.xcodeproj/project.pbxproj
@@ -309,7 +309,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvPerspective_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";
@@ -323,7 +323,7 @@
 				CINDER_PATH = ../../../../..;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = ocvPerspective_Prefix.pch;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				INFOPLIST_FILE = Info.plist;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

--- a/samples/ocvWarp/xcode/ocvWarp.xcodeproj/project.pbxproj
+++ b/samples/ocvWarp/xcode/ocvWarp.xcodeproj/project.pbxproj
@@ -314,7 +314,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = macosx;
@@ -331,7 +331,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				GCC_WARN_ABOUT_RETURN_TYPE = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/boost\"";
+				HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\"";
 				MACOSX_DEPLOYMENT_TARGET = 10.8;
 				SDKROOT = macosx;
 				USER_HEADER_SEARCH_PATHS = "\"$(CINDER_PATH)/include\" ../include ../../../include ../../../include/opencv2";

